include/Spectra/JDSymEigsBase.h

```diff
@@ -110,10 +110,10 @@ class JDSymEigsBase
     }
 
 protected:
-    virtual Matrix SetupInitialSearchSpace() const = 0;
```

Comment:
Can we add the const here again?

Reply:
I have removed the setting of the diagonal to the constructor and return the constaness :+1: